index.html

```diff
@@ -0,0 +1,114 @@
+          </div>
+        </nav>
+      </header>
+      <section id="event-intro" class="container-fullwidth">
+        <span class="pre-title">"Hello! Sharing World"</span>
+        <h1 class="title">Creative Commons Global Summit 2015</h1>
+        <p class="introduction">
+          A joyful celebration believing in the value of openness and 
+          sharing, creating a positive change with people from over 80 
+          countries is taking place in October, in Korea.
+        </p>
+        <article class="datetime-place">
+          <h2>2015.10.15(THU) ~ 16(FRI)</h2>
+          <div>
+            @ National Museum of Korea, Art Center Nabi and more
+          </div>
+        </article>
+      </section>
+    </main>
+    <section class="container-fullwidth" id="programs">
+      <h2>Main Program</h2>
+      <hr class="hr">
+      <ul id="programs-list">
+        <li id="program1" class="program-item" tabindex="0">
+          <img src="assets/icons/program1.png" alt="Lecture Icon">
+          <h3>Lecture</h3>
+          <span>
+            Listen to the speakers from various countries about
+            the messages of sharing and opening.
+          </span>
+        </li>
+        <li id="program2" class="program-item" tabindex="0">
+          <img src="assets/icons/program2.png" alt="CC Exhibition Icon">
+          <h3>CC Exhibition</h3>
+          <span>
+            Appreciate various creations applying CC license of
+            artists, organized from Art Center Nabi.
+          </span>
+        </li>
+        <li id="program3" class="program-item" tabindex="0">
+          <img src="assets/icons/program3.png" alt="Forum Icon">
+          <h3>Forum</h3>
+          <span>
+            Have the time to share your thoughts and opinions with
+            experts for each topic.
+          </span>
+        </li>
+        <li id="program4" class="program-item" tabindex="0">
+          <img src="assets/icons/program4.png" alt="Workshop Icon">
+          <h3>Workshop</h3>
+          <span>
+            Try creating your own work using open source license
+            rather than just watching.
+          </span>
+        </li>
+        <li id="program5" class="program-item" tabindex="0">
+          <img src="assets/icons/program5.png" alt="CC Ignite Icon">
+          <h3>CC Ignite</h3>
+          <span>
+            Get opportunities to network with CC affiliates around
+            the world, also after the summit.
+          </span>
+        </li>
+      </ul>
+      <button id="join-summit" class="mobile-visible" type="button">Join CC Global Summit 2015</button>
+      <button id="seewhole" class="desktop-visible">See The Whole Program</button>
+    </section>
+    <section id="speakers">
+      <h2>Featured Speakers</h2>
+      <hr class="hr">
+      <ul id="speakers-list"></ul>
+      <button id="all-speakers" type="button" onclick="javascript:void(0)" class="mobile-visible">
+        <span class="btn-more">More</span>
```

Comment:
- [ ] Images of featured speakers and partners sections are not visible on the page. Please make sure you use the correct path for the images.
 - To fix this replace your path `/%PUBLIC_URL%./assets/images/.....` with `./assets/images/......`

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/163255312-9b331675-b6d9-4661-b552-7817497f9161.png)